apps/geo/schema.py

```diff
@@ -107,6 +107,13 @@ class Meta:
     region_title = graphene.String(required=True)
     admin_level_title = graphene.String(required=True)
 
+    @staticmethod
+    def resolve_region_title(root, info, **kwargs):
+        return root.admin_level.region.title
+
+    def resolve_admin_level_title(root, info, **kwargs):
+        return root.admin_level.title
+
```

Comment:
This will create an N+1 query.

You can pass the queryset through get_geo_area_queryset_for_project_geo_area_type to annotate the region and admin_level title.